cmd/admin/admin.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/common/yaml"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/config"
+	etcdv3 "github.com/dubbogo/dubbo-go-proxy/pkg/remoting/etcd3"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+)
+
+var client *etcdv3.Client
+
+
+const (
+	// EtcdAddress etcd address
+	EtcdAddress   string = "127.0.0.1:2379"
```

Comment:
you may need to run go fmt.

Reply:
and why we hardcode the address and port here?

Reply:
ok, and i will ad config file for admin address and port